fem/restriction.hpp

```diff
@@ -57,6 +57,10 @@ class ElementRestriction : public Operator
    /// Compute MultTranspose without applying signs based on DOF orientations.
    void MultTransposeUnsigned(const Vector &x, Vector &y) const;
 
+   /// Compute MultTranspose by setting (rather than adding) element
+   /// contributions; this is a left inverse of the Mult() operation
+   void MultLeftInverse(const Vector &x, Vector &y) const;
```

Comment:
I don't see how the R operator has a left inverse — it has fewer rows than columns, and so it cannot have full column rank.

Reply:
Despite its name, `ElementRestriction` is more of a P matrix than a R matrix, it takes L-vectors to E-vectors and has more rows than columns. Is there a way to make the documentation clearer?

Reply:
Ok, that makes sense (I should have looked more closely at the rest of `restriction.hpp`) — I guess it's using the libCEED "ElemRestriction" terminology. There is some potential for confusion with the restriction that is given by `FiniteElementSpace::GetRestrictionMatrix`.